packages/gsm8k/src/aviary/envs/gsm8k/env.py

```diff
@@ -67,7 +67,7 @@ async def step(
                     Message(
                         content=(
                             "Must call one of the provided tools (calculator or"
-                            " check_answer)."
+                            " submit_answer)."
```

Comment:
```suggestion
                            f" {self.submit_answer.__name__})."
```

More tolerant to typos. Can you do the same for `calculator` above